experimental/dds/tree/src/PayloadUtilities.ts

```diff
@@ -65,16 +66,14 @@ export function comparePayloads(a: Payload, b: Payload): boolean {
 	}
 
 	// Special case IFluidHandles, comparing them only by their absolutePath
-	// Detect them using JavaScript feature detection pattern: they have a `IFluidHandle` field that is set to the parent object.
-	{
-		const aHandle = a as IFluidHandle;
-		const bHandle = b as IFluidHandle;
-		if (aHandle.IFluidHandle === a) {
-			if (bHandle.IFluidHandle !== b) {
-				return false;
-			}
-			return a.absolutePath === b.absolutePath;
+	if (fluidHandleSymbol in a) {
+		if (fluidHandleSymbol in b) {
```

Comment:
This is an example of code needing to compare handles, which is currently only possible using the internal (previously deprecated) APIs.